pkg/runtimeoptions/v1/api.proto

```diff
@@ -11,4 +11,7 @@ message Options {
 	// ConfigPath specifies the filesystem location of the config file
 	// used by the runtime.
 	string config_path = 2;
+	// Blob specifies an in-memory TOML blob passed from containerd's configuration section
```

Comment:
Just seeing this change.
I'm wondering why this would be TOML?

This struct already has a type url supplied, why wouldn't the body be the marshaled raw data?

This seems extremely CRI specific.

Reply:
I mean the marshaled version, such as proto or json.

Reply:
so per discussion..need an issue to align the type_url to logic specifying that said type defines the bytes in config_body if present...

Reply:
Opened https://github.com/containerd/containerd/issues/8246